app/controllers/container_image_controller.rb

```diff
@@ -1,5 +1,5 @@
 class ContainerImageController < ApplicationController
-  include ContainersCommonMixin
+  include Mixins::ContainersCommonMixin
   include Mixins::BreadcrumbsMixin
```

Comment:
I thought mixins was on the autoload paths?

Reply:
See above.

Reply:
yea, happy that mixin is now included via namespacing.
Having a nested folder that is also included was a little confusing for me.

Reply:
Yeah, honestly, I don't like the name but I was torn between somethings that used the namespace and others that did not and it was easier to make them consistent and use it.  Some have different types of behavior and perhaps ultimately, they should live in a different location but beyond the scope of making these files play nice with zeitwerk.